contrib/JavaGraphqlClient/src/test/java/com/example/graphql/client/JavaGraphqlClient/JavaGraphqlClientApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.example.graphql.client.JavaGraphqlClient;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class JavaGraphqlClientApplicationTests {
+
+	@Test
+	void contextLoads() {
+	}
+
+}
```

Comment:
could you add an example test here to show how things work?

Reply:
I will add the testcase